README.md

```diff
@@ -1,97 +1,49 @@
-# official frontend
+# Official Cryptic Frontend
 
 [![Build Status](https://travis-ci.org/cryptic-game/frontend.svg?branch=master)](https://travis-ci.org/cryptic-game/frontend)
 
-This is the official frontend of Cryptic.  
-https://play.cryptic-game.net/
+This is the official [Cryptic](https://play.cryptic-game.net/) frontend ([available at docker-hub](https://hub.docker.com/r/crypticcp/frontend/)). It has been created with the **Angular** framework using _TypeScript_, _HTML_ and [SASS](http://sass-lang.com/guide).  
 
-# Advice for Developers
+## Set up a development environment
 
-This web application is created with the **Angular** framework. The Framework uses the languages _TypeScript_ (JavaScript with typing), _HTML_ and _SCSS_. A fundamental understanding of these languages is important to understand the code. _SCSS_ or [_Sass_](http://sass-lang.com/guide) is a CSS preprocessor. You can find a tutorial [here](https://www.toptal.com/sass/theming-scss-tutorial).
+NodeJS and NPM are required to compile and start the application. Refer to [https://nodejs.org/de/](https://nodejs.org/de/) for download/installation instructions. To get help, run `ng --help` or check the [Angular documentation](https://angular.io/docs) / [Angular CLI documentation](https://github.com/angular/angular-cli/wiki).
 
-The framework also uses a **CLI** (_Comand Line Interface_). It helps creating and managing Angular projects fast.
+[Microsoft Visual Studio Code](https://code.visualstudio.com) is the recommended editor. Visual Studio Code (VSCode) provides a build-in debugger and a _git_ client.  
```

Comment:
Change this to a list and add IntelliJ Ultimate/Webstorm (or with a student license)

Reply:
Idk the tool, was it invluded before? Need to do some research. Can you add a suggestion.using the button to the top left? @MarcelCoding

Reply:
```suggestion
- [Microsoft Visual Studio Code](https://code.visualstudio.com) is the recommended editor. It provides a build-in debugger and a _git_ client.  
- IntelliJ Ultimate/Webstorm (paid or student license) 
```